package.json

```diff
@@ -41,22 +44,22 @@
     "request-promise": "^4.2.2"
   },
   "devDependencies": {
-    "babel-cli": "^6.26.0",
```

Comment:
Can you add back the babel CLI?
```javascript
"@babel/cli": "^7.4.3",
```

Reply:
My bad, I forgot to had it back during the dependencies upgrade process. 👍